gstd/gstd_element.c

```diff
@@ -335,6 +367,45 @@ gstd_element_internal_to_string (GstdElement * self, gchar ** outstring)
   g_free (properties);
 
   gstd_iformatter_end_array (self->formatter);
+
+
+  gstd_iformatter_set_member_name (self->formatter, "element_signals");
+  gstd_iformatter_begin_array (self->formatter);
+
+  signals = g_signal_list_ids (G_OBJECT_TYPE (self->element), &n_signals);
```

Comment:
Why don't we iterate on the signal lists instead of doing all the queries again?